src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -43,61 +42,53 @@ namespace SonarLint.VisualStudio.Integration.UnitTests
+            Action act = () => new BindingProcessImpl(null, slnBindOp, bindingConfigProvider, exclusionSettingsStorage, configurationPersister, sonarQubeService, logger);
             act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("bindingArgs");
 
-            // 3. Null solution binding operation
-            act = () => new BindingProcessImpl(validHost, bindingArgs, null, bindingConfigProvider, exclusionSettingsStorage);
+            // 2. Null solution binding operation
+            act = () => new BindingProcessImpl(bindingArgs, null, bindingConfigProvider, exclusionSettingsStorage, configurationPersister, sonarQubeService, logger);
             act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("solutionBindingOperation");
 
-            // 4. Null rules configuration provider
-            act = () => new BindingProcessImpl(validHost, bindingArgs, slnBindOp, null, exclusionSettingsStorage);
+            // 3. Null binding configuration provider
+            act = () => new BindingProcessImpl(bindingArgs, slnBindOp, null, exclusionSettingsStorage, configurationPersister, sonarQubeService, logger);
             act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("bindingConfigProvider");
 
-            // 5. Null exclusion settings storage
-            act = () => new BindingProcessImpl(validHost, bindingArgs, slnBindOp, bindingConfigProvider, null);
+            // 4. Null exclusion settings storage
+            act = () => new BindingProcessImpl(bindingArgs, slnBindOp, bindingConfigProvider, null, configurationPersister, sonarQubeService, logger);
             act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("exclusionSettingsStorage");
+
+            // 5. Null configuration persister
+            act = () => new BindingProcessImpl(bindingArgs, slnBindOp, bindingConfigProvider, exclusionSettingsStorage, null, sonarQubeService, logger);
+            act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("configurationPersister");
+
+            // 6. Null SonarQubeService
+            act = () => new BindingProcessImpl(bindingArgs, slnBindOp, bindingConfigProvider, exclusionSettingsStorage, configurationPersister, null, logger);
+            act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("sonarQubeService");
+
+            // 7. Null logger
+            act = () => new BindingProcessImpl(bindingArgs, slnBindOp, bindingConfigProvider, exclusionSettingsStorage, configurationPersister, sonarQubeService, null);
+            act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("logger");
         }
 
         [TestMethod]
         public async Task SaveServerExclusionsAsync_ReturnsTrue()
         {
             var bindingArgs = CreateBindCommandArgs(projectKey: "projectKey");
+            var logger = new TestLogger(logToConsole: true);
```

Comment:
Fix up existing tests - `logger` and `configurationPersister` are no longer supplied as services so they need to be created explicitly in the tests that need them.